### DISCUSSION TITLE
Update from CefSharp 103.0.9 to 107.1.40 results in empty web view in Wine expect URL, what am I missing?

### DISCUSSION THREAD

**Discussion by tarasis** [cefsharp/CefSharp#4572] [General]
Hi I'm trying to figure out what I need to install (if particular version of .net or .net core). I understand that you likely don't do support for Wine issues, and certainly not Playnite. But you might be able to provide insight that I need to try and fix the issue.

Playnite swapped from using CefSharp 103.0.9 in 10.3, to 107.1.40 in 10.4. In 10.3 it had a working Web view when Playnite was run in Wine (it currently only has .net 4.8 installed). However, when upgrading to Playnite 10.4-10.18 the web view is empty EXCEPT for the url. ( The change https://github.com/JosefNemec/Playnite/commit/af95d5c286a524f09b0ec92bf7a2a0931fb7350c )

This suggests to me that I am missing something in the Wine install that would help CefSharp's web view work.

Is there any particular packages I should install?

**Comment by amaitland** [comment]
> I'm trying to figure out what I need to install (if particular version of .net or .net core).

You can have a look at https://github.com/cefsharp/CefSharp#release-branches

> Is there any particular packages I should install?

If `M103` worked then you likely have everything required for `M107`.

Your best bet would be to see if there are any errors in the `CEF` log. 

- https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file
- https://cefsharp.github.io/api/113.1.x/html/P_CefSharp_CefSettingsBase_LogFile.htm (There is a command line arg listed)

https://github.com/cefsharp/CefSharp.MinimalExample is the `CefSharp` sample application, you can try test it directly.

**Reply by tarasis** [reply]
Thanks for the pointers @amaitland, very helpful.

I've downloaded and compiled the MinimalExample but I can't get it to run in Wine. 

```
[0904/232451.778:ERROR:crashpad_client_win.cc(534)] CreateProcess: File not found. (0x2)
[0904/232451.802:INFO:crash_reporting.cc(216)] Crash reporting enabled for process: browser
[0904/232451.962:ERROR:network_change_notifier_win.cc(221)] WSALookupServiceBegin failed with: 0
[0904/232452.005:WARNING:chrome_browser_cloud_management_controller.cc(88)] Could not create policy manager as CBCM is not enabled.
[0904/232452.973:INFO:crash_reporting.cc(216)] Crash reporting enabled for process: gpu-process
[0904/232453.271:ERROR:gpu_process_host.cc(956)] GPU process exited unexpectedly: exit_code=-1073740286
[0904/232453.271:WARNING:gpu_process_host.cc(1326)] The GPU process has crashed 1 time(s)
```

---

Playnite doesn't seem to have a log file for CefSharp.

---

Trying to run `cefyclient.exe` from [cef_binary_116.0.19+gc6a20bc+chromium-116.0.5845.141_windows64_client.tar.bz2](https://cef-builds.spotifycdn.com/cef_binary_116.0.19%2Bgc6a20bc%2Bchromium-116.0.5845.141_windows64_client.tar.bz2) crashes, I have downloaded the debug symbols for the release, but the program error dialog hasn't finished "Loading detailed information". 

---

Will need to sit and have a proper look at this, possibly custom compile playnite if it has logging of Cef disabled.
